x-pack/platform/test/fleet_multi_cluster/config.ts

```diff
@@ -12,7 +12,7 @@ import { RemoteEsProvider } from './services/remote_es/remote_es';
 
 export default async function ({ readConfigFile }: FtrConfigProviderContext) {
   const xpackFunctionalConfig = await readConfigFile(
-    require.resolve('@kbn/test-suites-xpack/functional/config.base')
+    require.resolve('../api_integration/config.ts')
```

Comment:
shouldn't this be `'../functional/config.base.ts'` like above?

Reply:
I checked the services, that are used in test:

```
  const security = getService('security');
  const retry = getService('retry');
  const remoteEs = getService('remoteEs' as 'es');
  const localEs = getService('es');
  const supertest = getService('supertest');
```

It looks like an API-integration test and if so, there is no need in loading page objects. api_integration config provides all the required FTR services (e.g. `es` or `supertest`). Do I miss something?

Reply:
You're right, we don't need page objects. It should be good if the tests are passing.